src/components/primitives/index.ts

```diff
@@ -42,7 +48,7 @@ export type { ISpinnerProps } from './Spinner';
 export { default as Heading } from './Heading';
 export type { IHeadingProps } from './Heading';
 
-export { Pressable } from './Pressable';
+export * from './Pressable';
```

Comment:
same should be applied here.

Reply:
Thank you for your feedback. I fixed it in the recent commit.